app/Models/UserReport.php

```diff
@@ -35,7 +35,7 @@ class UserReport extends Model
     use RoutesNotifications, Validatable;
 
     const POST_TYPE_REASONS = ['Insults', 'Spam', 'UnwantedContent', 'Nonsense', 'Other'];
-    const SCORE_TYPE_REASONS = ['Cheating', 'Other'];
+    const SCORE_TYPE_REASONS = ['Cheating', 'MultipleAccounts', 'Other'];
```

Comment:
Reporting multi account on score is fine but also need to be added to `availableOptions` in `report-reportable.tsx`